network-api/networkapi/wagtailpages/pagemodels/publications/publication.py

```diff
@@ -85,22 +76,34 @@ class PublicationPage(FoundationMetadataPageMixin, Page):
     )
     notes = RichTextField(
         blank=True,
+        features=['link', 'bold', 'italic']
     )
     contents_title = models.CharField(
         blank=True,
         default="Table of Contents",
         max_length=250,
     )
     content_panels = Page.content_panels + [
-        MultiFieldPanel([
-            FieldPanel('subtitle'),
-            FieldPanel('secondary_subtitle'),
-            FieldPanel('publication_date'),
-            ImageChooserPanel('hero_image'),
-            DocumentChooserPanel('publication_file'),
-            InlinePanel("authors", label="Author"),
-            FieldPanel("additional_author_copy"),
-        ], heading="Hero"),
-        FieldPanel('contents_title'),
-        FieldPanel('notes')
+        MultiFieldPanel(
+            [
+                FieldPanel("subtitle"),
+                FieldPanel("secondary_subtitle"),
+                FieldPanel("publication_date"),
+                ImageChooserPanel("hero_image"),
+                DocumentChooserPanel("publication_file"),
+                InlinePanel("authors", label="Author"),
+                FieldPanel("additional_author_copy"),
+            ],
+            heading="Hero",
+        ),
+        FieldPanel("contents_title"),
+        FieldPanel("notes"),
     ]
+
+    @property
+    def is_chapter_page(self) -> bool:
```

Comment:
none of our other code uses this kind of typing, and an `is_...` already tells us that the result is a boolean, so to keep things consistent, let's keep this a normal `def`.